src/scripts/GameController.ts

```diff
@@ -183,6 +182,25 @@ class GameController {
                     }
             }
 
+            if (App.game.underground.energy < App.game.underground.getMaxEnergy()) {
+                if ($('#mineModal').hasClass('show')) {
+                    switch (e.code) {
+                        case 'Digit1':
+                            ItemList['SmallRestore'].use();
+                            break;
+                        case 'Digit2':
+                            ItemList['MediumRestore'].use();
+                            break;
+                        case 'Digit3':
+                            ItemList['LargeRestore'].use();
+                            break;
+                        default: // any other key (ignore)
+                            return;
+                    }
+                    e.preventDefault();
+                }
+            }
+
```

Comment:
Could you move this up to be where the other shortcut keys are, you can also just make use of the `numKey` variable up there too.

We can also remove the check if they are already at max energy, as when you use the item it will check for them and also display a notification if they are already full.

Reply:
Done this as well.